.circleci/config.yml

```diff
@@ -2,73 +2,15 @@ version: 2.1
 orbs:
   node: circleci/node@5.2.0
 
-references:
-  executor: &executor
+jobs:
+  check:
```

Comment:
Since it appears CCI is a required check, it might be good to temporarily add a dummy config for a smoother transition.